.github/workflows/publish-rust-crate.yml

```diff
@@ -49,20 +49,20 @@ jobs:
         with:
           cargo-cache-key: cargo-test-publish-${{ inputs.crate_path }}
           cargo-cache-fallback-key: cargo-test-publish
-          toolchain: format, lint, test
+          toolchain: build, format, lint, test
           solana: true
 
       - name: Format
-        run: tsx ./scripts/rust.mts format "${{ inputs.crate_path }}"
+        run: npx tsx ./scripts/rust.mts format "${{ inputs.crate_path }}"
```

Comment:
`npx` isn't ideal because IIRC it will download the package on-demand even though it is actually part of the dependencies.

How would you feel about having a `pnpm rust` script instead? So here we'd be able to do:

```
run: pnpm rust format "${{ inputs.crate_path }}"
```

Reply:
I replaced `npx` by `pnpm` – would that work better? Perhaps it would be nicer not to add another script on `package.json` that only CI would use.